README.md

```diff
@@ -21,3 +21,37 @@ be available for use.
 # Running the image Locally
 After building the image, you can run it locally using the following command: <br>
 `podman run -p 8080:8080 MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080`
+
+## How to add the image to opendatahub/JupyterHub
+
+* Once you have pushed your image to an image registry that is accessible by your JupyterHub instance,
+you can use the `ImageStream` template in this repository at `openshift/oc-imagestream.yaml`.
+
+
+```yaml
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true" # <-- setting it true makes it visible to ODH/JupyterHub
+  name: s2i-custom-notebook # <-- Update the ImageStream name here
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - from:
+      kind: DockerImage
+      name: docker-registry.upshift.redhat.com/aicoe-notebooks/s2i-custom-notebook:latest # <-- Change to your container Image with tag
+    importPolicy:
+      scheduled: true
+    name: "latest"
+```
+
+* After updating the template file with your image details, you can run the following command to create a ImageStream in your JupyterHub Namespace:
+
+  `oc create -f openshift/oc-imagestream.yaml -n MY_JUPYTERHuB_NAMESPACE`
```

Comment:
I'd recommend suggesting `oc apply` since that is more versatile and generally preferred command